src/os/dispatch/dispatcher.js

```diff
@@ -0,0 +1,33 @@
+goog.module('os.dispatch.dispatcher');
```

Comment:
Yeah, it's a little wonky for sure because of the namespace overlap and the lack of it being a class. At first I was thinking I'd just have it be a class that extends EventTarget, but that broke things down the line with the Analyze tool. So I figured it'd be likely to break things I couldn't see too. It looks like Kevin's idea will work great though if I can get it to not break everything attached to "os" -- we'll see soon.